### PR TITLE
fix: apply vueMeta directly instead of delaying

### LINKED ISSUE #9453: [Bug Report] Theme colors don't change after mounted
### Environment
**Vuetify Version:** 2.1.6
**Last working version:** 2.1.5
**Vue Version:** 2.6.10
**Browsers:** Google Chrome
**OS:** Linux

### Steps to reproduce
Downgrade repo to vuetify@2.1.5 and mounted color is correct

### Expected Behavior
Mounted color changes

### Actual Behavior
Mounted color does not change and stays as default

### Reproduction Link
<a href="https://codesandbox.io/s/codesandbox-nuxt-8tbth" target="_blank">https://codesandbox.io/s/codesandbox-nuxt-8tbth</a>

### Other comments
Might be related to vue-meta changes: https://github.com/vuetifyjs/vuetify/commit/6d829efff05ff6d491a719b21f3ea88f5ce1aeaf

### LINKED ISSUE #9499: [Bug Report] Theme loading issue
### Environment
**Vuetify Version:** 2.1.7
**Last working version:** 2.1.4
**Vue Version:** 2.6.10
**Browsers:** Chrome 77.0.3865.120
**OS:** Windows 7

### Steps to reproduce
1. Open the attached reproduction link.
2. Wait for build.
3. Press the refresh icon in the embedded codesandbox browser.
4. See the theme color is display black until 1-2 seconds.
5. After fully load the page, then color is switched to the "red" primary color.

### Expected Behavior
Immediately display the page with the appropriate theme colors.

### Actual Behavior
The page theme color is black until 1-2 seconds. It is not so good.

### Reproduction Link
<a href="https://codesandbox.io/s/nuxtjs-vuetify-217-b0dfj" target="_blank">https://codesandbox.io/s/nuxtjs-vuetify-217-b0dfj</a>

### Other comments
Thanks in advance for the fix. :-) @DRoet and @johnleider.

### PR DESCRIPTION
**Pull request by DRoet**
fixes #9499
fixes #9453

## Description

Force vue-meta render by calling `refresh()` instead of waiting for `mounted()`
## Motivation and Context

Some extra discussions over here: https://github.com/nuxt/vue-meta/issues/481
## How Has This Been Tested?

visually
## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by johnleider** [comment]
Pulse check.

**Comment by DRoet** [comment]
Still needs help, I'm not sure how to delay the `vue-meta` stuff untill the `created()` hook since the current theme stuff is in the `beforeCreate()` hook

**Comment by oanstein** [comment]
Although I have found a way to avoid the bug (https://github.com/vuetifyjs/vuetify/issues/9453), I am still interested in a correct fix ...

**Comment by amritk** [comment]
Hmm I tested the fix but I'm getting
```typescript
 ERROR  [Vue warn]: Error in beforeCreate hook: "ReferenceError: document is not defined"

(found in <Root>)
```
for
```typescript
    created () {
        const themes = {
            amber: '#EFB00A',
            'argent-grey': '#5D5D5D',
            'blue-grey': '#607D8B',
            'coffee-brown': '#543B25',
            'baywatch-red': '#DB261E',
            'deep-red': '#B71C1C',
            'fire-orange': '#E64A19',
            'indigo-blue': '#003382',
            'kelly-green': '#008940',
            'steel-blue': '#0E5E9E',
            'burnt-umber': '#842424',
            'pigment-indigo': '#4B0082'
        }
        const theme = themes[this.league.theme?.permalink ?? 'steel-blue']
        this.$vuetify.theme.themes.light.primary = { base: theme }
        this.$vuetify.theme.themes.dark.primary = theme
    }
```

**Comment by DRoet** [comment]
this fix is incomplete so I'm not surprised it doesn't work 😅

**Comment by amritk** [comment]
Any updates on this? Or workarounds?

**Comment by johnleider** [comment]
Pulse check.

**Comment by DRoet** [comment]
I can't put any time into this and I don't use vue-meta.

**TLDR: vue-meta stuff needs to initialize in the `created()` hook and not `beforeCreate()`, but is currently nested in the theme stuff so not easy to change.**

**Comment by amritk** [comment]
I have found a workaround in the meantime:
```typescript
created () {
    // Temporary until it gets fixed
    this.$vuetify.theme.themes.light.primary = { base: '#0E5E9E' }
    this.$vuetify.theme.themes.dark.primary = { base: '#0E5E9E' }
    this.$vuetify.theme.applyVueMeta23()
}

mounted () {
    this.$meta().refresh()
}
```

**Comment by johnleider** [comment]
Closing due to inactivity. We are about to move into LTS with v2.3 and this hasn't had any traction.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
